virtcontainers/api.go

```diff
@@ -129,27 +129,27 @@ func StartSandbox(sandboxID string) (VCSandbox, error) {
 	defer unlockSandbox(lockFile)
 
 	// Fetch the sandbox from storage and create it.
-	p, err := fetchSandbox(sandboxID)
+	sandbox, err := fetchSandbox(sandboxID)
```

Comment:
Please use `s` to be consistent with other places.

Reply:
There are still some `sandbox` not `s` in functions like `StatusSandbox`.May be I should change them to `s`

@bergwolf  What do you think?

Reply:
@jshachm Yes, please. Thanks!